draft-ietf-mls-protocol.md

```diff
@@ -852,13 +856,13 @@ each node MUST be the parent of its predecessor.
 
 ~~~~~
 struct {
-    DHPublicKey ephemeral_key;
+    PublicKey ephemeral_key;
```

Comment:
Since we're moving to a more KEM-like framework, it might be good to also change this from `ephemeral_key` to something like `encapsulated_key`.

Also, while you're working here, please go ahead and increase the size of the `ciphertext` field.

Reply:
Also, the reason I had `DH` there was to distinguish from `SignaturePublicKey`.  Just having `PubilcKey` vs. `SignaturePublicKey` seems less clear.  What about `KEMPublicKey` or `HPKEPublicKey`?

Reply:
I removed this as we can point to the HPKE draft to get how the HPKECiphertext is internally organized.
There is still a sentence saying what the argument of HPKE Encrypt (node_secrets) are.